### ISSUE TITLE
Asus aura and corsair link

### ISSUE THREAD

**Issue by Bchap9** [antonpup/Aurora#546]
You should add support to control rgb headers, graphics cards, motherboards through Asus aura, and cpu coolers through corsair link. Love the program

**Comment by antonpup** [comment]
I have an Asus Aura motherboard, I don't think they've released an SDK for it yet.

As for Corsair, I am pretty sure that "Corsair Link" is just a name, everything RGB goes through CUE, which Aurora already has support for.

**Comment by BobVul** [comment]
There's an SDK in beta now, with release planned at Computex (end of May): https://rog.asus.com/tag/aura-sync-sdk/

**Comment by caw1** [comment]
i´ve got a reply for that issue: i translated it from german : "thank you for your request. Such an enhancement for our graphics or Motherboards is not planned yet. The lighting can only be controlled by the AURA Software.

Marco Janssen
Customer Service Center
ASUS Computer Germany

**Comment by brainbug89** [comment]
@caw1 maybe you should sent Mr. Marco Janssen this link: https://rog.asus.com/articles/coming-soon/the-poseidon-gtx-1080-ti-and-other-new-rog-products-debut-at-jtr-outshine-the-competition/ (last paragraph)

**Comment by caw1** [comment]
Hi,

Finally i've found the Link: 
https://www.asus.com/campaign/aura/global/SDK.html

:) 

If some of you have the Time and the Knowledge to implement that in Aurora Many of us were so happy :)

**Comment by stbatuhan** [comment]
Well there is no support for corsair link actually. I'm using h100i gtx and not compatible of course. Well it's not good at their software too.

**Comment by Fma965** [comment]
Is this something that can be done? i would absolutely love to get my keyboard and mobo in sync :) 

https://github.com/opticalgenesis/AuraCUE 

not sure if the above link is helpful or not?

**Comment by ersan** [comment]
Corsair case lighting is separate from CUE, unfortunately - it all runs through Corsair LINK instead, which is terrible software.  I would absolutely love if Aurora could support Corsair RGB fans and Lighting Nodes but I don't know how possible it is.

There is a third party program called SIV (http://rh-software.com/) that is capable of controlling the LEDs but I don't know to what extent and it is closed source.  Here is a post detailing how to control the LEDs with SIV, if it helps: http://forum.corsair.com/forums/showthread.php?p=892388 (and how to control Corsair DIMM LEDs: http://forum.corsair.com/forums/showthread.php?p=888742)

**Comment by caw1** [comment]
are there any News about the implementing of the Asus Aura SDK?

**Comment by simon-wh** [comment]
@caw1 No, the plan, at least for me is to look at it when it comes to 0.7.0. Which won't be started until after v0.6.1 is stable.

**Comment by caw1** [comment]
@simon-wh , thats the right way. im happy to hear that there are some further plans in your mind :)

**Comment by OminiData** [comment]
i am not sure if it is possible but aquacomputer  support would be an awesome feature
https://github.com/aquacomputer/plugin_sdk

**Comment by deivater** [comment]
Support for corsair link would be sick. Can´t imagine, how nice it will looks, with my Corsair node pro and RGB Fans. If this feature will be added, I will send you a paypal donation, because this is the feature, I´m waiting for :)  Aurora is awesome.

**Comment by Fma965** [comment]
not sure how useful this is but have you seen https://github.com/DarthAffe/RGB.NET?

**Comment by simon-wh** [comment]
@Fma965 I had a look at that before and was considering using it, but it would require rewriting a decent amount of Aurora and isn't really needed, I'm currently reworking the device system which will allow for proper integration of devices outside of keyboards and mice, so this will be supported once I've finished that.

**Comment by alwe2710** [comment]
What's the status of the Aura implementation? Will it get a feature of Aurora? I would be so lucky about it

**Comment by darknesdragon** [comment]
Hey @simon-wh does the full aura support include the aura support for ASUS ROG GL553VD laptop?
the laptop is not using AURA Sync but it is listed using AURA RGB, I hope the SDK have something for the laptop.

I would love to see if I can customize the keyboard lighting from aurora
